python/ray/autoscaler/local/defaults.yaml

```diff
@@ -42,11 +34,21 @@ auth:
     # Optional if an ssh private key is necessary to ssh to the cluster.
     # ssh_private_key: ~/.ssh/id_rsa
 
-# Leave this empty.
-head_node: {}
-
-# Leave this empty.
-worker_nodes: {}
+available_node_types:
+    ray.head.default:
+        resources: {}
+        min_workers: 0
+        max_workers: 0
+        # Leave this empty
+        node_config: {}
+    ray.worker.default:
+        resources: {}
+        ## NOTE: Typically for local clusters, max_workers == len(worker_ips).
+        min_workers: 0
+        max_workers: 0
```

Comment:
why max_workers = 0?

Reply:
I took it out for consistency with the other defaults yamls.
0 because that's the way it way it was before.

It was probably that way before to force people to consciously set the number of nodes they want to use in their local cluster.